src/main/java/mate/academy/hibernate/relations/service/impl/MovieServiceImpl.java

```diff
@@ -1,16 +1,23 @@
 package mate.academy.hibernate.relations.service.impl;
 
+import mate.academy.hibernate.relations.dao.MovieDao;
 import mate.academy.hibernate.relations.model.Movie;
 import mate.academy.hibernate.relations.service.MovieService;
 
 public class MovieServiceImpl implements MovieService {
+    private final MovieDao movieDao;
+
+    public MovieServiceImpl(MovieDao movieDao) {
+        this.movieDao = movieDao;
+    }
+
     @Override
     public Movie add(Movie movie) {
-        return null;
+        return movieDao.add(movie);
     }
 
     @Override
     public Movie get(Long id) {
-        return null;
+        return movieDao.get(id).orElse(null);
```

Comment:
Returning null from Optional nearly makes no sense.
Better to throw relevant exception with proper message

```suggestion
        return movieDao.get(id).orElseThrow(new EntityNotFoundException("Movie with id " + id + " is not found));
```